documentation/explainers/price-feed-identifiers.md

```diff
@@ -0,0 +1,77 @@
+# Price feed identifiers
+
+This document explains how price feed identifiers are configured and how they are used.
+
+## What is an identifier
+
+An identifier is the term we use to refer to an underlying asset whose price a derivative tracks. Examples of underlying assets
+are the price of Bitcoin in US dollars, the spot price of gold, or the daily high temperature in New York City.
+
+## Configuration file
+
+Price feed identifiers are configured in [identifiers.json](`https://github.com/UMAprotocol/protocol/blob/master/core/config/identifiers.json`).
+This json file contains a list of entries for each supported identifier, with two bits of configuration information:
+1. `dappConfig`: configuration for the Sponsor dApp.
```

Comment:
nit: directly referencing the sections might slightly improve the UX here:
```
1. `dappConfig`: configuration for the Sponsor dApp. See the [Sponsor dApp](#sponsor-dapp) section for more details.
2. `uploaderConfig`: configuration for UMA's price feed uploader. See the [Uploader](#uploader) section for more details.
```

Reply:
Done.